src/rabbit_variable_queue.erl

```diff
@@ -273,6 +273,7 @@
           msg_store_clients,
           durable,
           transient_threshold,
+          index_max_size,
```

Comment:
I find this name confusing. What about `queue_index_embedding_threshold ` or even `queue_index_embed_msgs_below `?